core-web/libs/data-access/src/lib/dot-properties/dot-properties.service.spec.ts

```diff
@@ -70,6 +71,78 @@ describe('DotPropertiesService', () => {
         req.flush(apiResponse);
     });
 
+    describe('getKey', () => {
+        it('should return the value of a key if it exists in featureConfig', () => {
+            const key = 'existingKey';
+            const value = 'existingValue';
+            service.featureConfig = { [key]: value };
+            service.getKey(key).subscribe((result) => {
+                expect(result).toEqual(value);
+            });
```

Comment:
we should use the `done` callback to ensure that this test is doing the assertion

```javascript
it('should return the value of a key if it exists in featureConfig', (done) => {
            const key = 'existingKey';
            const value = 'existingValue';
            service.featureConfig = { [key]: value };
            service.getKey(key).subscribe((result) => {
                expect(result).toEqual(value);
                done();
            });
```

Reply:
This applies to all similar tests